privacyidea/api/lib/prepolicy.py

```diff
@@ -690,6 +690,7 @@ def check_max_token_user(request=None, action=None):
     :return: True otherwise raises an Exception
     """
     ERROR = "The number of tokens for this user is limited!"
+    ERROR_ACTIVE = "The number of active tokens for this user is limited!"
     params = request.all_data
     user_object = get_user_from_param(params)
```

Comment:
There is one problem here: If an admin requests ``/enable/<serial>``, we do not have a ``user`` in the ``request.all_data``, so ``user_object.login`` is empty and the maximum number of tokens is not checked.

Reply:
You are right. That's pitty.

Reply:
OK, I opened #1652 to keep track of the issue.